pyproject.toml

```diff
@@ -83,5 +83,38 @@ artifacts = ["/backend/fastrtc/templates", "*.pyi"]
 packages = ["/backend/fastrtc"]
 
 [tool.ruff]
+src = ["demo", "backend/fastrtc"]
 target-version = "py310"
 extend-exclude = ["demo/phonic_chat", "demo/nextjs_voice_chat"]
+
+[tool.ruff.format]
+exclude = ["*.pyi"]
+quote-style = "double"
+indent-style = "space"
+skip-magic-trailing-comma = false
+line-ending = "auto"
+
+[tool.ruff.lint]
+select = [
+  "E",
+  "F",
+  "W",
+  "Q",
+  "I",
+]
+
+# These can be turned on when the framework is more mature (Too many errors right now)
+exclude = [
+  "D",
```

Comment:
Yes I agree. I am working on adding docstrings and an API reference right now but even still I don't think we would want to document every method for every class

Reply:
Agreed, it is a bit too agressive.